src/backend/index_builder.rs

```diff
@@ -23,14 +23,40 @@ pub trait IndexBuilder: QuotedBuilder {
 
         write!(sql, " ON ").unwrap();
         if let Some(table) = &create.table {
-            table.prepare(sql, self.quote());
+            self.prepare_table_ref(table, sql);
         }
 
         self.prepare_index_type(&create.index_type, sql);
 
         self.prepare_index_columns(&create.index.columns, sql);
     }
 
+    /// Translate [`TableRef`] into SQL statement.
+    fn prepare_table_ref(&self, table_ref: &TableRef, sql: &mut SqlWriter) {
+        self.prepare_table_ref_common(table_ref, sql);
+    }
+
+    fn prepare_table_ref_common(&self, table_ref: &TableRef, sql: &mut SqlWriter) {
```

Comment:
We talk with you about it. MySql and SQLite don't support schema, only database. I think implementation this method should be backend specific.